### ISSUE TITLE
xcodeproj is ancient version

### ISSUE THREAD

**Issue by uli-objectbox** [krzysztofzablocki/Sourcery#789]
Hi,

I just noticed that xcodeproj is pinned to 4.3.1, while the tuist repository is already at 7.x.

Is there a reason for that? Currently Sourcery fails for me with Xcode projects that contain Swift Package Manager packages, as it does not support those object types.

I've managed to change the files in Pods to just ignore these types and that made it work for me for now, but before I go and try to fix this properly, I wanted to ask whether there is some deeper reason why we're not on current xcodeproj?

Cheers,
-- Uli

**Comment by uli-objectbox** [comment]
I did try just bumping the version, and it seems there are other Pods that depend on the same modules at different versions, e.g. PathKit 1.0. Did nobody get around to following them all, or is it impossible to resolve them right now?

**Comment by krzysztofzablocki** [comment]
I think we just didn't look at this in a while since we didn't run into issues before, would you like to try to make a pr to update those dependencies to newer variants?

**Comment by uli-objectbox** [comment]
So xcodeproj requires PathKit 1.0. If I up Sourcery's requirement to 1.0, that still leaves Stencil, which still references 0.9.1: 

```
[!] CocoaPods could not find compatible versions for pod "PathKit":
  In Podfile:
    PathKit (= 1.0.0)

    Stencil (= 0.13.1) was resolved to 0.13.1, which depends on
      PathKit (~> 0.9.0)

    xcodeproj (from `https://github.com/tuist/XcodeProj.git`, tag `7.1.0`) was resolved to 7.1.0, which depends on
      PathKit (~> 1.0.0)
```

Changing that dependency to 1.0.0 as well in a fork makes CocoaPods happy, but StencilSwiftKit gets unhappy (even when bumped to 2.7.2). Apparently that's not up-to-date either.

**Comment by krzysztofzablocki** [comment]
we are on newest version now
